package.json

```diff
@@ -27,8 +20,6 @@
     "node-fetch": "^2.6.1",
     "object-hash": "^3.0.0",
     "proto3-json-serializer": "^1.0.0",
-    "protobufjs": "7.2.3",
```

Comment:
I'm pretty sure we still need `protobufjs` as a runtime dependency, we use it in many places in the code.

https://github.com/googleapis/gax-nodejs/blob/6d17ba2b459112cf6506262b765f3f3468a9cbcd/src/fallback.ts#L19